src/Controls/src/Core/Platform/Android/Extensions/ToolbarExtensions.cs

```diff
@@ -143,9 +143,9 @@ public static void UpdateBarTextColor(this AToolbar nativeToolbar, Toolbar toolb
 			// We need to reset the toolbar text color to the default color when it's unset
 			if (_defaultTitleTextColor == null)
 			{
-				var a = TintTypedArray.ObtainStyledAttributes(nativeToolbar.Context?.GetThemedContext(), null, Resource.Styleable.Toolbar, Resource.Attribute.toolbarStyle, 0);
-				_defaultTitleTextColor = a.GetColorStateList(Resource.Styleable.Toolbar_titleTextColor);
-				a.Recycle();
+				var a = nativeToolbar.Context?.GetThemedContext ().GetStyledAttributesForStyleable (nameof(Resource.Styleable.Toolbar), Resource.Attribute.toolbarStyle);
+				_defaultTitleTextColor = a?.GetColorStateList(Resource.Styleable.Toolbar_titleTextColor);
+				a?.Recycle();
```

Comment:
Can more of this code move into Java?

I'm thinking the `Recycle()` call, and maybe the Java method could return a `ColorStateList`?